### ISSUE TITLE
[Quest] Suggestion: Add Wrought Iron quest

### ISSUE THREAD

**Issue by richardhendricks** [GTNewHorizons/NewHorizons#1770]
#### Which modpack version are you using?
1.5.0.7
#### What do you suggest instead/what changes do you propose?
Currently the questline goes from steam items straight to the bricked blast furnace.  I propose adding a new quest under Tier 1 Steam that is after Macerator V1.0 and Compressing Stuff - Better Tools.  This quest would tell players to take crushed iron of some type and smelt it to get nuggets (maybe require crushed ore of each iron type so that players can discover that not all crushed ore produces the same number of nuggets? Or a mention in the quest book), and then turn the iron nuggets into Wrought Iron by smelting again, and using the compressor to make ingots, and then a couple of wrought iron tools.

Also, as a suggestion, perhaps make high pressure steam blocks use wrought iron instead of steel? And upgradable from low pressure versions? Seems like many people prefer to just skip over them, but making them something you can easily selectively upgrade as needed (say macerators, since they are used in many processes), would make players more likely to use the machines.

Thanks for the modpack!

**Comment by Dream-Master** [comment]
Upgrade the bronze machines to Steel (or Wrough Iron in your case) i have this idea in the past too but people mean that bronze machines can be dissasembled later and you get the stuff back.
@Drobac @Technus @bigbass1997 @Bryfer @MineAnPlay what you think abut this idea

**Comment by richardhendricks** [comment]
Thanks for the consideration. Just finished making the bricks for my first BBF, so excited!

**Comment by bigbass1997** [comment]
Quest for retrieving Wrought Iron Ingots for sure; but I don't know about a quest for tools, maybe make that part an optional quest. I honestly don't bother making wrought iron tools usually, as making wrought iron early on is long and annoying.

I've never bothered using any of the High Pressure versions of machines, so I can't say one way or another for that idea.

**Comment by Giovanni-NL** [comment]
I like all aspects of this idea.  Using a quest to teach players how to make wrought iron is a good idea, and fits well with the rest of the quest on the page.  As bigbass already mentioned, tool quests should be optional.  And upgradable machines is something I'd like to see indeed.

**Comment by Dream-Master** [comment]
But there is a quest how you mae quick wrough iron in the arc furnace in 32 LV Age. Did we need more ?

**Comment by richardhendricks** [comment]
Well, you can make wrought iron pretty early in Steam, once you have crushed ores or impure dusts and have a compressor.  It would be a nice benefit to players to know about it earlier since your tools would last longer (cooking utensils, saws, files, etc).  Plus it would be a good opportunity to expose players to different iron ores providing different amounts of nuggets - only some ores double (20 nuggets per ore), some only give 60% more (12 nuggets per ore).

Even though the bronze steel machines are so slow, I am reluctant to spend steel on creating high pressure steam ones because of the time it takes to process - each machine requires a fairly large steel investment, and doesn't advance tiers.  So making high pressure steam versions using wrought iron would make them cheap enough that I think more people would use them. 

Plus they exist in a virtuous cycle with the macerators making crushed ores that can be smelted twice and then compressed into ingots and turned into plates all with simple hoppers used to transport the goods, so the simple automation can be left running while players are exploring/mining.  As the wrought iron plates are made, machines could be upgraded/replaced to make this process faster. This takes the TiCon smeltery out of the system too, making it easier to get finished goods vs trying to keep the lava tanks filled while processing hundreds of nuggets to make vanilla iron.

**Comment by bigbass1997** [comment]
> But there is a quest how you mae quick wrough iron in the arc furnace in 32 LV Age. Did we need more ?

As I recall you need wrought iron much before you ever make this machine or even get to that quest. So yes a quest that explains the process to make wrought iron without the Arc Furnace is what we need. This is a very commonly asked question from new players.

**Comment by Giovanni-NL** [comment]
^ that.  You need wrought iron to make your first LV machine hull.  So a quest to make wrought iron just before reaching LV makes perfect sense.

**Comment by Drobac** [comment]
Might be a good idea as people just skip HP completely as its a waste to ise steel on it instead of LV machines

**Comment by Dream-Master** [comment]
Ok sure i can rewrite the whole recipe and make using the Bricked Blast Furnace Wrough iron too.

**Comment by Dream-Master** [comment]
@Drobac Did HP machines need Bronze Machines in the recipes that people can upgrade machines ?

**Comment by Bryfer** [comment]
I belive using wrought iron to make HP steam machines is a bad idea. It is easier to make Wrought Iron than to make bronze in low tiers. This most likely means that there will be suggestions later to skip the LP steam machines completely, or change the LP steam machines to optional quests and make HP steam machines main quests.
Don't forget how easy it is to make wrought iron, and iron is more common than copper and tin.

However. I agree that the HP steam machines are pretty useless since they are using the same steel that allows you to unlock electric age. - But since HP steam machines are optional in the questing, I dont think a change here is really nessesary. or do we really need to extend steam tiers before people can unlock LV (electric) tiers?

Making wrought iron in Steam tier is pretty slow aswell. since you need to put every nugget inside the furnance. and then compress it again.
instead of making multiple (vanilla) furnances to do this, why not make 4-8 EBFs as intended? - and let players decide if they want to do the optional quests or rush LV tier themselves?

**my conclusion; keep HP machines as it is. (Steel
Add an quest for wrought iron (thought it already existed in steam tier)**

> I had an idea to use another material than wrought iron at first, but considering everything useful requires an EBF, this would require alot of work, and rethinking of alot of recipies.

~~Also, what other kinds of metals / alloys could be better to use for the HP steam machinery if a change is needed?
Wouldn't it be better to find somekind of suiting alloy you could craft HP steam machines with instead? Cupro Nickel for example, has an similar (bit lower) melting point compared to Steel and Bronze.
Another dust I could suggest wold be Black Bronze (3x copper 1x gold 1x silver). however in the current verision it requires the EBF to be crafted.~~
```

**Comment by Dream-Master** [comment]
"Add an quest for wrought iron (thought it already existed in steam tier)"

Thats true but it is a faster producing wrough iron not a quest how you make it in the first place.

**Comment by richardhendricks** [comment]
I see your point, it is easier to babysit or automate wrought iron production vs bronze, but to make any wrought iron a compressor is needed, so it is gated by having the bronze steam machines.  I can see more people doing the optional quests to at least upgrade commonly used machines like the macerator or steam boiler if they used wrought iron instead of steel. 

 At my current level I have two small bronze steam boilers and two bronze steam macerators. Everything else is just a single bronze machine. Even with two macerators I spent quite a bit of time waiting for them to macerate ores/clay or wheat.  Even though I now have two BBFs, I am probably not going to upgrade any of them just because of the time for steel. 

If there is a concern people would just build HP machines, make the LP machines part of the recipe for the HP machines - remove the current HP recipes, and make them craftable by putting wrought iron plates around a LP machine. This way users would still be incentivized to build the LP machines and use them and upgrade them one by one to the HP versions, if they really want to.  There would still be a reasonable tradeoff between using the iron to upgrade to HP or saving it to make into steel, which I think is a good tension/option for the player.

If anything, this suggestion doesn't extend steam tier, but actually would accelerate getting through it since you could upgrade the steam machines before hitting ULV.

**Comment by Bryfer** [comment]
Since Dream made a voting pool for this subject, I belive it would be a good idea to include it here aswel..

https://strawpoll.com/brdspede
Current votes when this post was made: (3rd of august 2017)
![image](https://user-images.githubusercontent.com/22825059/28941184-7ba5b80a-7897-11e7-8cc0-4b75e7e29c2d.png)

**Comment by Dream-Master** [comment]
This vote was a first reaction how people think about it after it was discaussed in the staff channel.

**Comment by richardhendricks** [comment]
I am OK with boilers remaining steel.  It would make for a natural progression:

bronze -> need more crushing -> add HP wrought iron machines -> now need more steam -> add HP steel boilers

Plus the steel boilers can still be used in Tier 2, to generate ULV power, right?

**Comment by Dream-Master** [comment]
https://strawpoll.com/brdspede

**Comment by Dream-Master** [comment]
![Uploading image.png…]()

**Comment by Dream-Master** [comment]
![image](https://user-images.githubusercontent.com/2970688/28994843-19ea956c-79d8-11e7-9880-a408384d97db.png)
a early version of making wrought iron before compressor

**Comment by richardhendricks** [comment]
I love it!  Thanks!

**Comment by Dream-Master** [comment]
remove the nugget to tiny pile recipe because it was to easy.

**Comment by richardhendricks** [comment]
Working great, replaced and upgraded all my bronze items, except for the boilers of course.  I did add a new HP boiler so I can keep up, and another layer to my Iron Tank to keep steam on stock. 

Also figured out I can use clean dusts in the bricked blast furnace for another 50% in ingots, thanks!  May be time to double my BBF to 4.  OF course, then pollution will be a major problem, as well as feeding them charcoal...

I did kinda sorta accidentally plant the 7x7 redwood, so I am set for wood for a long long time. Fun times!
